keycloak.yaml

```diff
@@ -109,6 +109,10 @@ subpackages:
           # an initContainer
           cp -r ${{targets.destdir}}/usr/share/java/keycloak/* ${{targets.contextdir}}/opt/bitnami/keycloak
 
+          # Properly symlink the providers directory after installation
+          rm -rf ${{targets.contextdir}}/opt/bitnami/keycloak/providers/
+          ln -sf /usr/share/java/keycloak/providers ${{targets.contextdir}}/opt/bitnami/keycloak
```

Comment:
I suspect this will re-introduce the same issue since everything under `/opt/bitnami/keycloak` is copied from the image into the volume (https://github.com/bitnami/charts/blob/bb05f9f24ef535ec72ac760529b15f2b7bd67254/bitnami/keycloak/templates/statefulset.yaml#L108)

Reply:
Ah, it could be. So we should make a new subpackage for `cp` it?